_data/gerbi.json

```diff
@@ -43,6 +43,17 @@
         "https://orcid.org/0000-0002-3328-9467",
         "https://github.com/Tom-TBT"
       ]
+    },
+    {
+      "@type": "Person",
+      "givenName": "Johannes",
+      "familyName": "Soltwedel",
+      "jobTitle": "Software Developer",
+      "description": "Johannes Soltwedel is a physicist by training and found his way into bio-image world and the life sciences through his PhD at the department for radio-oncology at the Center for Proton Therapy at the University Hospital Dresden, Germany. Then and during his subsequent position as bio-image analysist at Physics of Life (TU Dresden, Germany) he learned the intricacies of propper bio-image analysis, software development and the value of good research data management throughout a multitude of different collaborative projects. He is currently living in Dresden with his family and enjoys an ever-growing house plant collection, everything related to tinkering and 3d-printing as well as a good hike.",
```

Comment:
analysist -> analyst

(this is more usual)

Reply:
This is why we :heart: @pwalczysko!

Reply:
:)